docs/0002-choose-a-pagination-strategy-and-implementation.md

```diff
@@ -6,7 +6,7 @@ Date: 2019-12-31
 
 ## Status
 
-Provisional
+Accepted
```

Comment:
It's weird to be PRing an ADR kind of -- like, it's not accepted right now... but it will be if it's approved and merged. maybe "switch to accepted" should be a PR step for ADRs. We're allowed to have different pull request templates for different purposes, so maybe that would be something good to do for this kind of thing.

Reply:
The weirder thing is that if someone looks at the branch and sees the status is "accepted," it's possible they could be misled. We don't have super long lived branches where that'd really be an issue, but this is just a little incongruous.

Reply:
Oh woops this was the previous ADR. Nevermind! This one _should_ be accepted. I remember thsi commit now.